packages/components/src/components/FormInput/FormInput.stories.tsx

```diff
@@ -12,9 +12,6 @@ import { ControlledFormInput } from "./ControlledFormInput";
 const meta: Meta<typeof FormInput> = {
   title: "Components/Form/FormInput",
   component: FormInput,
-  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-  // @ts-ignore Table has a required prop which confuses Storybook here.
-  subcomponents: { ControlledFormInput },
 };
```

Comment:
This was causing an error on the docs page so I've removed it. There seems to be a separate issue where the props table doesn't seem to be working in our storybook that we should look into.